src/math/vec2.h

```diff
@@ -0,0 +1,95 @@
+//
+// Created by ajer on 16.12.18.
+//
+
+#pragma once
+#include "pch.h"
```

Comment:
precompiled header should be included in source file, and only if you use libraries included there